packages/nimble-tokens/source/styledictionary/properties/colors.json

```diff
@@ -152,6 +152,10 @@
       "value": "#356D54ff",
       "type": "color"
     },
+    "PowerGreenText": {
```

Comment:
I'm not sure if we should use the word "Text" in the colors.json file since all the others seem to be the names of colors. 

So should it be like the below as I see it is a lighter version of `PowerGreen`?
```suggestion
    "PowerGreenLight": {
```

However, looking into the linked issue, it seems the suggested toke name is 
```suggestion
    "PowerGreenText100": {
```

Reply:
As you've mentioned `PowerGreenText100` is the suggested name. But considering that we'll be appending the opacity at the end when we use a percent of the given color like 10, 30 or 110 and not for 100 percent, I deferred adding it.

Having this comment open to get thoughts from other reviewers as well.

Reply:
Chatted with @NIbokeefe and for the styledictionary token lets go with `PowerGreenDark50` (representing 50 opaque and no transparency as all tokens in this file are `xff` for alpha).

He shared this image which shows the token name concepts of Opaque (against a background, i.e. Light or Dark) and Transparency (the actual alpha used):

![image](https://github.com/user-attachments/assets/a1809db2-620f-4969-93b6-82db87b4b846)

Reply:
I've updated the name to PowerGreenDark50. Thanks for sharing that info on naming the base tokens, @rajsite.